BASE/src/ServerTelemetryChannel/Implementation/TransmissionPolicy/ApplicationLifecycleTransmissionPolicy.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation.TransmissionPolicy
```

Comment:
dont think its really worthy as these are internals anyway.

Reply:
We have 5 classes that inherit TransmissionPolicy. (I'm adding a 6th in my new PR).
I had a hard time finding the existing clrasses because they were all in the root. 

I've moved them all to a unique subfolder and updated the namespace to match.